src/utils/account.js

```
@@ -16,22 +16,36 @@ export const extractPublicKey = (passphrase) => {
   return undefined;
```

These retuened `undefined` are essentially silent failures. I think we should add a logger here so we don't overlook the broken features.

---

Same goes for the other two convertors.